BookingsAPI/Bookings.UnitTests/Controllers/HearingsController/Helpers/RequestBuilder.cs

```diff
@@ -31,6 +33,17 @@ public static BookNewHearingRequest Build()
             return request;
         }
 
+        private static List<LinkedParticipantRequest> LinkedParticipantsBuilder(List<ParticipantRequest> participants)
```

Comment:
Is this reflective of a proper request; i.e. are you sure the participants are individuals?

Reply:
A new filtered variable created with claimants that are linked.